docs/sources/storage/_index.md

```diff
@@ -305,6 +308,71 @@ This guide assumes a provisioned EKS cluster.
    `bucket_name` variable.
 
 
+### IBM Cloud Object Storage
+
+```yaml
+schema_config:
+  configs:
+    - from: 2020-10-01
+      index:
+        period: 24h
+        prefix: loki_index_
+      object_store: "cos"
+      schema: v11
+      store: "boltdb-shipper"
+
+storage_config:
+  cos:
+    bucket: <bucket>
+    endpoint: <endpoint>
+    api_key: <api_key_to_authenticate_with_cos>
+    region: <region>
+    service_instance_id: <cos_service_instance_id>
+    auth_endpoint: <iam_endpoint_for_authentication>
+```
+
+The role should have a policy with the following permissions attached.
```

Comment:
IMO, it is sufficient to indicate that the IAM Writer role is needed, let's avoid such details which might confuse people.

Reply:
If they want to have more advanced access, they should refer to IAM.

Reply:
Remove the policy json